Registry/registry.noahmp

```diff
@@ -110,3 +110,20 @@ state integer   pgsxy      ij      -       1      -     rh
 
 # Gecros crop model only
 state    real    gecros_state i{gecros}j  -       1      Z     rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)     "gecros"      "gecros state array"         "-"
+
+#Noah-MP Irrigation
+state    real   IRFRACT    ij      -       1      -     i012rd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRFRACT"     "Irrigation fraction"            ""
+state    real   SIFRACT    ij      -       1      -     i012rd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "SIFRACT"     "Sprinkler Irrigation fraction"            ""
+state    real   MIFRACT    ij      -       1      -     i012rd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "MIFRACT"     "Micro Irrigation fraction"            ""
+state    real   FIFRACT    ij      -       1      -     i012rd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "FIFRACT"     "Flood Irrigation fraction"            ""
+state  integer  IRNUMSI    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRNUMSI"     "Sprinkler Irrigation Event Count" ""
+state  integer  IRNUMMI    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRNUMMI"     "Micro Irrigation Event Count" ""
+state  integer  IRNUMFI    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRNUMFI"     "Flood Irrigation Event Count" ""
+state    real   IRWATSI    ij      -       1      -     r                                                           "IRWATSI"     "Sprinkler Irrigation Amount for the Event"     "m"
+state    real   IRWATMI    ij      -       1      -     r                                                           "IRWATMI"     "Micro Irrigation Amount for the Event"     "m"
+state    real   IRWATFI    ij      -       1      -     r                                                           "IRWATFI"     "Flood Irrigation Amount for the Event"     "m"
+state    real   IRSIVOL    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRSIVOL"     "Sprinkler Irrigation Water Accumulated"     "mm"
+state    real   IRMIVOL    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRMIVOL"     "Micro Irrigation Water Accumulated"     "mm"
+state    real   IRFIVOL    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRFIVOL"     "Flood Irrigation Water Accumulated"     "mm"
+state    real   IRELOSS    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRELOSS"     "Sprinkler Evaporation Loss Accumulated"     "mm"
+state    real   IRRSPLH    ij      -       1      -     i02rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)    "IRRSPLH"     "Sprinkler Evaporation Loss Accumulated"     "Joules m^-2"
```

Comment:
Should the new variables be packaged?

Reply:
They are 2d, but ideally yes we need to check if that is safe to do.